src/main/java/CLIOutput.java

```diff
@@ -19,16 +16,18 @@ public static void outputBoard (Board board, boolean symbol) {
             }
         }
 
-        for (Piece piece : pieceList) {
-            outputBoard[piece.getCoords()[0]][piece.getCoords()[1]] = piece.toString();
-        }
-
-        if (!symbol) {
-            for (int i = 0; i < pieceList.size(); i++) {
-                Piece piece = pieceList.get(i);
-                System.out.println(piece.getType());
+        for (Piece[] pieceList : pieceArray) {
+            for (Piece piece: pieceList) {
+                if (piece!=null) {
+                    if (symbol) {
+                        outputBoard[piece.getCoords()[0]][piece.getCoords()[1]] = getPieceSymbol(piece);
+                    } else {
+                        outputBoard[piece.getCoords()[0]][piece.getCoords()[1]] = getPieceLetter(piece);
+                    }
+                }
             }
         }
+
```

Comment:
Please be consistent in your usage of constant values, i.e. in some places Board.WIDTH / Board.HEIGHT appear but in others 8 is used when the former still applies.